reference/catalogue.v1.yaml

```diff
@@ -0,0 +1,276 @@
+      x-tags:
+        - catalogue
+      properties:
+        ean:
+          type: string
+          description: |
+            The unique EAN for this product.
+        type:
+          type: string
+          description: |
+            Type of product
+          enum:
+            - physical
+            - digital
+            - blended
+        status:
+          type: string
+          description: |
+            Status of product
+          enum:
+            - active
+            - not-for-sale
+        name:
+          type: string
+          description: A short name
+        subjectCodes:
+          type: array
+          description: The governement recognised subject codes this product relates to.
+          items:
+            type: string
+        levels:
+          type: array
+          description: |
+            Governmeent recognised levels the product relates to.
+          items:
+            type: string
+        models:
+          type: array
+          description: |
+            Business models that are available for this product to be sold under.
+          items:
+            $ref: '#/components/schemas/BusinessModel'
+        trialAccessUrl:
+          type: string
+          description: Optional trial / preview URL.
+        defaultAccessUrl:
+          type: string
+          description: The default url used to access the product after selection or purchase.
+        shortDescription:
+          type: string
+          description: |
+            Short description for display on search pages or other marketing blocks.
+        longDescription:
+          type: string
+          description: |
+            Long description for display on a product page.
+        media:
+          type: object
+          description: A collection of media for use in the marketplace.
+          properties:
+            mainThumbnailUrl:
+              $ref: '#/components/schemas/Media'
+            productImageUrls:
+              type: array
+              description: A collection of image urls for a product.
+              items:
+                $ref: '#/components/schemas/Media'
+            productVideoUrls:
+              type: array
+              description: A collection of video urls for a product.
+              items:
+                $ref: '#/components/schemas/Media'
+            productPdfUrls:
+              type: array
+              description: A collection of PDF links for a product.
+              items:
+                $ref: '#/components/schemas/Media'
+        relatedProducts:
+          type: array
+          description: |
+            An array of related product ID's (for display alongside a product in a marketplace).
+          items:
+            type: string
+        bundledProducts:
+          type: array
+          description: A link to any other products that are included in this one if it is a package / bundle.
+          items:
+            type: string
+      required:
```

Comment:
the property I miss is the list of streamcode that this product contains. If you "just" have the EAN the product is defined, but you have no clue what is in there.  It is the most important link to the content for reporting progress and meta data.

Often a EAN contains several streams ( example you can access material for next and previous year). The stream (or method) could also be part of another product, that is created to sell in a different way. 

There is an n-to-n relation between a product and a method.

Can we add: array of string: [ called streams or methods or streamcode]

Reply:
I think we may be using different words for the same thing (I acknowledge that a parent `method` or equivalent is missing).  Can you provide an example - perhaps one for Noordhoff, Thieme and Malmberg so we all have the same understanding?

Reply:
I've added the object as discussed, but am need to check internally if it makes sense from our perspective - I don't feel we model it this way, but can easily be wrong :D